src/egl/drivers/dri2/egl_dri2.h

```diff
@@ -59,6 +59,7 @@
 #include <system/window.h>
 #include "platform_android_gralloc_drm.h"
 #include <hardware/gralloc1.h>
```

Comment:
there is no include for gralloc1 in current Mesa, this is perhaps not rebased on top of current tree?